packages/fast-components-react-msft/src/flipper/flipper.tsx

```diff
@@ -75,6 +80,34 @@ class Flipper extends Foundation<FlipperHandledProps, FlipperUnhandledProps, {}>
 
         return coercedProps;
     }
+
+    private renderNextGlyph(): React.ReactNode {
+        if (this.props.direction === FlipperDirection.next) {
```

Comment:
what if we generated the path based on direction instead of the entire SVG?

Reply:
Then we could simply have a single function to generate the glyph

Reply:
Sounds good.